core/dbt/contracts/graph/unparsed.py

```diff
@@ -66,13 +66,45 @@ class Docs(JsonSchemaMixin, Replaceable):
     show: bool = True
 
 
+class AdditionalPropertiesMixin:
```

Comment:
I think this actually exists in `dev/marian-anderson` now (sorry, I've been busy since you opened this and should've clarified!) if you rebase/merge in, you can import it from `dbt.contracts.util`.